db/postgres/connection.go

```diff
@@ -414,7 +447,7 @@ func (d *DB) CommitTransaction() error {
 // pgx.
 func (d *DB) RollbackTransaction() error {
 	if d.tx == nil {
-		return nil
+		return errors.New("cannot commit a transaction that does not exist")
```

Comment:
please make this into an error in the errors lib (this facilitates error checking)